src/swap/components/SwapSettingsSlippageToggle.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import { cn } from '../../styles/theme';
+import type { SwapSettingsSlippageToggleReact } from '../types';
+
+export function SwapSettingsSlippageToggle({
+  className,
+}: SwapSettingsSlippageToggleReact) {
+  const [slippageMode, setSlippageMode] = useState<'Auto' | 'Custom'>('Auto');
```

Comment:
So it's either Auto or Custom..... so this is a boolean.

It feels here should be some flavour of boolean in this dev experience.

Reply:
Yeah..I'm going to update this to a boolean called customSlippage!